src/styles/actor/npc/_index.scss

```diff
@@ -317,6 +313,11 @@ $faded-color: #7a7971;
                 }
             }
         }
+
+        @import "inventory";
+        @import "spells";
+        @import "effects";
+        @import "../red-action-boxes";
```

Comment:
Was this moved to a deeper level of nesting for a technical reason?

Reply:
I'm not aware of other potential reasons, but yes: in this case it was because legacy styling was at higher specificity.